docs/docs/applying-filters.md

```diff
@@ -143,16 +149,20 @@ class FiltersForm extends React.Component {
         </div>
         <div className="form-group">
           <div className="col-sm-offset-3 col-sm-9">
-            <a className="btn btn-success" onClick={() => this.props.onClear()}>
+            <a className="btn btn-default" onClick={() => this.props.onClear()}>
```

Comment:
Could you please do NOT use arrow functions inside JSX, It could trigger a memory leak